server/historian/packages/historian-base/src/services/restGitService.ts

```diff
@@ -81,6 +82,11 @@ export class RestGitService {
 			);
 			defaultHeaders.Authorization = `Basic ${token.toString("base64")}`;
 		}
+
+		// We set the flag only for ephemeral containers
+		if (this.isEphemeralContainer) {
+			defaultHeaders["Is-Ephemeral-Container"] = this.isEphemeralContainer;
```

Comment:
Can we reuse `Constants.isEphemeralContainer` here?

Reply:
Done